pkgs/data/sgml+xml/stylesheets/xslt/docbook-xsl/default.nix

```diff
@@ -22,6 +22,15 @@ let
           stripLen = "1";
         })
 
+        # Fix reproducibility by respecting generate.consistent.ids in indexes
+        # https://github.com/docbook/xslt10-stylesheets/pull/88
+        # https://sourceforge.net/p/docbook/bugs/1385/
+        (fetchpatch {
+          url = "https://github.com/docbook/xslt10-stylesheets/commit/07631601e6602bc49b8eac3aab9d2b35968d3e7a.patch";
+          sha256 = "0igfhcr6hzcydqsnjsd181h5yl3drjnrwdmxcybr236m8255vkq3";
+          stripLen = "1";
```

Comment:
The stripLen is here because the main directory in the tarball is the xsl directory (renamed).